lib/integrations/awsoidc/deployservice.go

```diff
@@ -557,6 +558,21 @@ func upsertCluster(ctx context.Context, clt DeployServiceClient, clusterName str
 			ClusterName:       aws.String(clusterName),
 			CapacityProviders: requiredCapacityProviders,
 			Tags:              resourceCreationTags.ToECSTags(),
+		}, func(o *ecs.Options) {
+			o.Retryer = retry.NewStandard(func(so *retry.StandardOptions) {
+				so.MaxAttempts = 10
+				so.MaxBackoff = time.Minute
+				// Retry if an error is a missing ECS service-linked role.
```

Comment:
I would also add this link to this comment
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using-service-linked-roles.html#create-slr

Reply:
added